strings/base_iterator.h

```diff
@@ -126,7 +126,7 @@ namespace winrt::impl
 
```

Comment:
Pre-existing: `n - it` (the `operator-(difference_type n, fast_iterator it)` defined on lines 122-125 above) is completely bogus, and should be struck.

Reply:
While you're right, I fear at this point it would be an API break. So I'll leave the final call to @kennykerr.

Reply:
If Casey says its bogus, I think we should probably fix/remove it. Just include a test for whatever the correct behavior should be, if any.